### ISSUE TITLE
Unknown reloc type 264 and some decode failures on linux-arm64 nativeaot collection

### ISSUE THREAD

**Issue by kunalspathak** [dotnet/runtime#98996]
```
[17:35:58] ERROR: Unknown reloc type 264
[17:35:58] ERROR: Unknown reloc type 264
[17:35:58] ERROR: Decode Failure Left@ offset 1ded06917cc
```

**Comment by filipnavara** [comment]
264 would be `R_AARCH64_MOVW_UABS_G0_NC`. That's definitely not produced by the managed NativeAOT compiler unless there's memory corruption. I doubt it would be in native code either since many of those newer relocations are not widely supported.

Is there some more info? Executable / object files where it happened?

**Comment by kunalspathak** [comment]
These are related to the TLS relate ones that I added recently. Their relocation need to be handled in the superpmi tool. I handled it in https://github.com/dotnet/runtime/pull/98840, but there was missing `break` in between which was not setting the flag that they are handled.

https://github.com/dotnet/runtime/blob/bff39aa121dc5b1f2527c0a04078f5a68d0d37b7/src/coreclr/inc/corinfo.h#L3392-L3395

**Comment by filipnavara** [comment]
Fixed in #99008
